ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/_cq_dialog/.content.xml

```diff
@@ -211,7 +208,8 @@
                             <shortDescription
                               jcr:primaryType="nt:unstructured"
                               sling:resourceType="cq/gui/components/authoring/dialog/richtext"
-                              fieldLabel="Short Description"
+                              fieldLabel="Tooltip"
```

Comment:
Tooltip - by definition - The tooltiptext class holds the actual tooltip text. It is hidden by default, and will be visible on hover .
So - we should use `Short description` only